### PR TITLE
[14.0][OU-ADD] website_event_registration_free_text: Merged to website_event_questions

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT33820

please @pedrobaeza @willytal review this

### CODE REVIEW

File: openupgrade_scripts/scripts/website_event_questions/14.0.1.1/pre-migration.py [hunk 1]
```diff
@@ -27,3 +45,6 @@ def migrate(env, version):
             ),
         ],
     )
+
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website_event_questions/14.0.1.1/pre-migration.py]: Don't add empty lines inside a method

**Comment by CarlosRoca13** [comment]
Change done

**Comment by pedrobaeza** [comment]
@CarlosRoca13 It's `[OU-ADD]`

@legalsylvain this one is not an Odoo core module to be included in the migration issue.

**Comment by legalsylvain** [comment]
> @legalsylvain this one is not an Odoo core module to be included in the migration issue. 

Sorry, my bad. Fixed.

**Comment by CarlosRoca13** [comment]
Changed

**Review by MiquelRForgeFlow — APPROVED** [review]

File: openupgrade_scripts/scripts/website_event_questions/14.0.1.1/pre-migration.py [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+        """UPDATE event_question
+        SET question_type = "text_box"
+        WHERE free_text = True
+        """,
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """INSERT INTO event_registration_answer
+            (question_id, registration_id, value_text_box)
+        SELECT eaf.registration_id, eaf.question_id, eaf.answer
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website_event_questions/14.0.1.1/pre-migration.py]: The order field is twisted according the specification above.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: openupgrade_scripts/scripts/website_event_questions/14.0.1.1/pre-migration.py (hunk 2 above)
Comment (CarlosRoca13) [reply openupgrade_scripts/scripts/website_event_questions/14.0.1.1/pre-migration.py]: Yes sorry, changed

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**